<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleCal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="basic_group_box">
      <property name="title">
       <string>基本计算</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="percent_group_box">
      <property name="title">
       <string>百分比计算</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="relative_group_box">
      <property name="title">
       <string>相对计算</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
